enhancements/installer/connected-assisted-installer.md

```diff
@@ -209,6 +211,16 @@ that approach cannot be supported, users can proceed by providing an
 extra host (4 hosts to build a 3 node cluster, 6 hosts to build a 5
 node cluster, etc.).
 
+There was a question about how the agent is secured:
```

Comment:
will do once we have the whole story in place. we're currently discussing / designing it with SD & security teams.